Product/Production/Adapters/General/CONNECTAdminGUI/src/main/resources/log4j.properties

```diff
@@ -1,8 +1,16 @@
 # Root logger option
-log4j.rootLogger=ALL, stdout
+log4j.rootLogger=DEBUG, stdout, R
 
 # Direct log messages to stdout
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
-log4j.appender.stdout.Target=System.out
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
+
+log4j.appender.R=org.apache.log4j.RollingFileAppender
+log4j.appender.R.File=C:/delete_my/logs/AdminGUI.log
```

Comment:
Is this new logger being added specifically for this PR?  This path will fail in Linux.